tests/drivers/build_all/sensor/spi.dtsi

```diff
@@ -285,3 +285,11 @@ test_spi_bmi08x_gyro: bmi08x@24 {
 	gyro-hz = "1000_116";
 	gyro-fs = <1000>;
 };
+
+test_spi_tmag5170: tmag5170@25 {
+	compatible = "ti,tmag5170";
+	reg = <0x25>;
```

Comment:
You need to add an extra line to `cs-gpios` in `tests/drivers/build_all/sensor/app.overlay` for the added CS GPIO.

Reply:
Thanks, solved